src/web/components/elements/YoutubeBlockComponent.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { css } from 'emotion';
 
+import { videoTracking as gaRecord } from '@root/src/web/browser/ga/ga';
+import { record as ophanRecord } from '@root/src/web/browser/ophan/ophan';
```

Comment:
This is a take it or leave it because I don't have the fill picture but do we need to rename these? I think using `as` can be useful in certain situations (which this might be) but is best avoided because it breaks continuity and can hide results when searching